src/Components/TableHead.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class TableHead extends Component {
```

Comment:
You can use `React.PureComponent` or `Functional Components` to handle those stateless components
ref:
https://reactjs.org/docs/components-and-props.html#functional-and-class-components
https://reactjs.org/docs/react-api.html#reactpurecomponent